tests/test_cmdparse.py

```diff
@@ -153,26 +160,33 @@ def test_getOption(self, cmd):
         opt, is_inverse = cmd.get_option('not-there')
         assert (None, None) == (opt, is_inverse)
 
+        opt, is_inverse = cmd.get_option('--list')
+        assert (opt_append['name'], False) == (opt.name, is_inverse)
+
 
     def test_parseDefaults(self, cmd):
         params, args = cmd.parse([])
         assert False == params['flag']
         assert 5 == params['num']
+        assert [] == params['list']
 
     def test_parseShortValues(self, cmd):
-        params, args = cmd.parse(['-n','89','-f'])
+        params, args = cmd.parse(['-n','89','-f', '-l', 'foo', '-l', 'bar'])
         assert True == params['flag']
         assert 89 == params['num']
+        assert ['foo', 'bar'] == params['list']
 
     def test_parseLongValues(self, cmd):
-        params, args = cmd.parse(['--rare-bool','--num','89', '--no-flag'])
+        params, args = cmd.parse(['--rare-bool','--num','89', '--no-flag',
+                                  '--list', 'flip', '--list', 'flop'])
         assert True == params['rare']
         assert False == params['flag']
         assert 89 == params['num']
+        assert ['foo', 'bar', 'flip', 'flop'] == params['list']
 
     def test_parsePositionalArgs(self, cmd):
-        params, args = cmd.parse(['-f','p1','p2','--sub-arg'])
```

Comment:
I changed this in a627702 to give `cmd.parse` a list of recognized options. `gnu_getopt` picks up on these later-placed options now.